modulo4/Projeto - IWFS Sistema Bancário/src/index.ts

```diff
@@ -0,0 +1,233 @@
+import express, { Request, Response } from 'express';
+import cors from 'cors';
+import { User, users, Extrato } from './data';
+
+const app = express();
+const PORT:number = 3003;
+let usersList:User[] = users;
+const dataAtual = new Date();
+const anoAtual:number = dataAtual.getFullYear();
+const diaAtual:number = dataAtual.getDate();
+const mesAtual:number = dataAtual.getMonth() + 1;
+
+app.use(express.json());
+app.use(cors());
+
+app.post("/users", (req:Request, res:Response) => {          
+    try{
+        
+        const body: User = {
+            name: req.body.name,
+            cpf: req.body.cpf,
+            data: req.body.data,
+            saldo: 0
+        };
+
+        const idade:number = anoAtual - Number(req.body.data.split("/", 3)[2]);
+
+        if(idade > 18){
+            res.statusCode = 400;
+            throw new Error("Usuário precisa ter no mínimo 18 anos.");    
+        };
+
```

Comment:
Aqui na criação de usuário, sempre vai cair nesse erro porque o sinal de maior está incorreto, seria ao contrário, se a idade for menor que 18 anos cairá no erro, e não maior, beleza?

Então dessa forma sempre cai no erro